pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -419,6 +420,43 @@ func (j *jivaVolumeOptions) preupgrade(pvName, openebsNamespace string) error {
 	return nil
 }
 
+func scaleDownTargetDeploy(name, namespace string) error {
+	klog.Infof("Scaling down target deploy %s in %s namespace", name, namespace)
+	deployObj, err := deployClient.WithNamespace(namespace).Get(name)
+	if err != nil {
+		return err
+	}
+	pvLabelKey := "openebs.io/persistent-volume"
+	pvName := deployObj.Labels[pvLabelKey]
+	controllerLabel := "openebs.io/controller=jiva-controller," +
+		pvLabelKey + "=" + pvName
+	var zero int32
+	deployObj.Spec.Replicas = &zero
+	_, err = deployClient.WithNamespace(namespace).Update(deployObj)
+	if err != nil {
+		return err
+	}
+	podList := &corev1.PodList{}
+	for i := 0; i < 60; i++ {
```

Comment:
Better to add comment like , `wait for 5 min for the target deployment to scale down`

Reply:
Done.